src/Storage/ContentRequest/Listing.php

```diff
@@ -176,7 +176,9 @@ protected function runGroupSort($results)
         if (isset($taxGroup) && $taxGroup !== null) {
             foreach ($grouped as &$group) {
                 usort($group, function ($a, $b) use ($resultTaxOrders) {
-                    return $resultTaxOrders[$a->getId()] - $resultTaxOrders[$b->getId()];
+                    $aOrder = isset($resultTaxOrders[$a->getId()]) ? $resultTaxOrders[$a->getId()] : 0;
+                    $bOrder = isset($resultTaxOrders[$b->getId()]) ? $resultTaxOrders[$b->getId()] : 0;
+                    return $aOrder - $bOrder;
```

Comment:
I don't suppose you want to amend the commit to add a newline before that `return` per PSR-2 do you? :wink:

Reply:
Is that a PSR-2 thing? Sure, I can amend.